src/components/votingV2/votingV2.js

```diff
@@ -29,11 +34,51 @@ class VotingV2 extends React.Component {
     this.setState({ votingModeEnabled: !this.state.votingModeEnabled });
   }
 
+  handleCloseOnboarding() {
+    localStorage.setItem('closedVotingOnboarding', true);
+    this.setState({ showVotingOnboarding: false });
+  }
```

Comment:
`handleCloseOnboarding` and `state.showOnboarding` should IMO be inside the generic `<Onboarding />` component. You just need to pass `key='showVotingOnboarding'` to use as the `localStorage` key.